version.php

```diff
@@ -28,7 +28,9 @@
 // is a database change in the course of development.  It is used
 // internally to determine when a database upgrade is needed.
 //
-$v_database = 470;
+
+$v_database = 472;
```

Comment:
Are you skipping because of my PR? 471

Reply:
Don't worry about my backport as I can fix the conflicts from this.

Reply:
@sjpadgett , @adunsulag 
note no need to increment this in the patching branches. it's ok to do it, but not used, so also ok not to do it :)

Reply:
Had a conflict so I just bumped it to the higher version of the conflict.  Sounds like it doesn't really matter.